### PR TITLE
feat: add list of account exceptions

### PR DESCRIPTION
**Pull request by gelouko**
# Description
Solution proposal for issue #4200

### CODE REVIEW

File: tools/c7n_org/scripts/orgaccounts.py [hunk 1]
```diff
@@ -37,7 +37,9 @@
     '-f', '--output', type=click.File('w'),
     help="File to store the generated config (default stdout)")
 @click.option('-a', '--active', default=False, help="Get only active accounts", type=click.BOOL)
-def main(role, ou, assume, profile, output, regions, active):
+@click.option('-ex', '--exceptions', multiple=True,
```
Comment (kapilt) [inline tools/c7n_org/scripts/orgaccounts.py]: how about renaming this to skip or ignore, exceptions is a pretty nebulous term, especially in the context of policy management/execution.

Comment (gelouko) [reply tools/c7n_org/scripts/orgaccounts.py]: Better, indeed! Changed!

**Comment by gelouko** [comment]
@kapilt signed!

File: tools/c7n_org/scripts/orgaccounts.py [hunk 2]
```diff
@@ -119,7 +121,7 @@ def get_sub_ous(client, ou):
     return results
 
 
-def get_accounts_for_ou(client, ou, active, recursive=True):
+def get_accounts_for_ou(client, ou, active, recursive=True, ignoredAccounts=[]):
```
Comment (kapilt) [inline tools/c7n_org/scripts/orgaccounts.py]: generally its bad practice to ever use a mutable as a default argument for a function, as that variable is retained

ie.

```python
def foo(x=[]):
     x.append(1)
     print(x)

foo()
# prints [1]
foo()
# prints [1, 1]
```

while we're not mutating the arg here, let's just use an empty tuple `ignoreAccounts=()` which is immutable.

**Pull request merged**